### PR TITLE
Add TCK utility tests

### PR DESCRIPTION
**Pull request by ayoho**
Brings over the tests for the `TokenUtils` class that's used to create JWTs used throughout the TCK.

### CODE REVIEW

**Comment by ayoho** [comment]
@sberyozkin Here are the utility tests like we talked about. Let me know if I missed anything. I also made some minor updates to a couple pom.xml files to reference the parent `org.eclipse.microprofile:microprofile-tck-bom`; that was the reference I was missing that allows the version to be omitted from the dependencies in the `tck/pom.xml` file.

**Comment by sberyozkin** [comment]
Thanks @ayoho

**Comment by sberyozkin** [comment]
@ayoho It works OK, thanks. have one question only

File: tck/src/test/resources/suites/tck-base-suite.xml [hunk 1]
```diff
@@ -19,16 +19,22 @@
...
             <define name="excludes">
+                <include name="utils-extra" description="Additional utility tests" />
             </define>
             <run>
                 <include name="base-groups" />
                 <exclude name="excludes" />
             </run>
         </groups>
         <classes>
+            <class name="org.eclipse.microprofile.jwt.bridge.tck.util.TokenUtilsTest" />
```
Comment (sberyozkin) [inline tck/src/test/resources/suites/tck-base-suite.xml:34]: Should we really make these utility tests part of the TCKs that the implementations will have to pass ? My understanding these utility tests will only run during the TCK builds ?

Comment (sberyozkin) [reply tck/src/test/resources/suites/tck-base-suite.xml:34]: @ayoho Never mind, we do it in MP JWT too, kind of makes sense to run them too to make sure the token generation is sound before the actual TCK runs

**Review by sberyozkin — APPROVED** [review]
Thanks @ayoho, please keep moving more TCK tests in the next few weeks :-)

**Pull request merged**